common/changes/office-ui-fabric-react/snaps_2019-07-11-22-31.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think you can get rid of this file since it's the old way of doing change files.